app/models/purl.rb

```diff
@@ -124,6 +118,10 @@ def purl_virtual_object
     @purl_virtual_object ||= PurlVirtualObject.new(public_cocina:)
   end
 
+  def cocina_service_response_body
+    @cocina_service_response_body ||= purl_cocina_service.response_body.presence || '{}'
+  end
+
   # Normalize the role text to use consistent capitalization and remove trailing punctuation.
   def format_role(role_element)
     role_element.text.strip.capitalize.sub(/[.,:;]+$/, '').tr('|', '')
```

Comment:
Is this worth putting into the gem? Presumably we don't want this weird punctuation in any roles, although I didn't see data like this when I was working on it

Reply:
Oh, gotcha. Makes sense